### PR TITLE
Security fixes

### PR DESCRIPTION
**Pull request by Ebtoulson**
[card](https://rentgroup.atlassian.net/browse/SRV-6131)

- addresses CWE-494 and CWE-94
- fixed build status badge
- addressed rubocop issues and added rubocop check to ci workflow

### CODE REVIEW

File: spec/ops/config_spec.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+require 'spec_helper'
+
+RSpec.describe Ops::Config do # rubocop:disable Lint/EmptyBlock
+end
```
Comment (Stratus3D) [inline spec/ops/config_spec.rb:4]: Why was this file added?

Comment (Ebtoulson) [reply spec/ops/config_spec.rb:4]: because the file was blank and rubocop complained. I tried to disable the check and then re-enable it but it still complained so I added an empty block. I would haved added tests here if I knew what this code was supposed to be doing but I don't really know why the original authors decided to use a `hash` with `method_missing` if all possible options were known.

**Review by Stratus3D — APPROVED** [review]

File: spec/ops/config_spec.rb (hunk 1 above)
Comment (Stratus3D) [reply spec/ops/config_spec.rb:4]: I didn't realize it was an existing empty file. Strange that it wasn't populated but you aren't making it worse.

**Review by pasha — APPROVED** [review]
